test/Ruby/testdata/bundleShow

```diff
@@ -1,3 +1,4 @@
 Gems included by the bundle:
   * pkgOne (1.0.0)
   * pkgTwo (2.0.0)
+Use `bundle info` to print more detailed information about a gem
```

Comment:
This is a test case that is causing a failure in v1. I had this in my diff to make sure v2 could parse this and want to keep it in the PR.